test/make.sh

```diff
@@ -73,17 +82,25 @@ function docker() {
 # This function runs 'terraform validate' and 'terraform fmt'
 # against all directory paths which contain *.tf files.
 function check_terraform() {
-  set -e
-  echo "Running terraform validate"
-  find_files . -not -path "./test/fixtures/shared/*" -name "*.tf" -print0 \
-    | compat_xargs -0 -n1 dirname \
-    | sort -u \
-    | compat_xargs -t -n1 terraform validate --check-variables=false
+  local rval=125
+  # fmt is before validate for faster feedback, validate requires terraform
+  # init which takes time.
```

Comment:
FYI, linting is considerably slower now due to `terraform init`, but no slower than integration.   Total time for all 3 pipeline jobs is unchanged.

Reply:
Is this a required change? Linting should be *much* faster than integration and have minimal requirements.

Reply:
Yes.  In 0.12 `terraform validate` does not work without `terraform init` having run.

Reply:
I see, thanks. We might eventually want to have a separate pure linting step (and put validation into a "unit test" of sorts).

Reply:
Good idea

Reply:
Kitchen-Terraform runs `terraform validate` as part of the workflow, so we could drop this entirely in favour of running it during integration tests.